src/Common/MainController.h

```diff
@@ -315,7 +315,7 @@ public slots:
 
     QScopedPointer<Geo::IpToLocationResolver> ipToLocationResolver;
 
-    QList<Recorder::JamRecorder> jamRecorders;
+    QList<Recorder::JamRecorder *> jamRecorders;
```

Comment:
@pljones , I checked your commits and I can't see the *delete*s to these dynamic allocated JamRecorders. QList will not delete the dynamic allocated elements.